pinot-core/src/main/java/org/apache/pinot/core/data/table/TableResizer.java

```diff
@@ -357,4 +372,88 @@ public Comparable extract(Record record) {
       }
     }
   }
+
+  public static class fullIntermediateResult extends IntermediateRecord {
```

Comment:
You can directly change the `IntermediateRecord` to include the record

Reply:
Class name should be capitalized

Reply:
Now directly extend IntermediateRecord

> You can directly change the `IntermediateRecord` to include the record